src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const basePrice = 40;
+  const threeDaysDiscount = 20;
+  const threeDays = 3;
+  const sevenDaysDiscount = 50;
+  const sevenDays = 7;
+
+  if (days >= sevenDays) {
```

Comment:
Consider using `const` instead of `let` for the `basePrice`, `threeDaysDiscount`, `threeDays`, `sevenDaysDiscount`, and `sevenDays` variables, as their values are not intended to change. This can help prevent unintentional reassignments and improve code clarity.